packages/client/src/sync/fetcher/accountfetcher.ts

```diff
@@ -276,6 +277,8 @@ export class AccountFetcher extends Fetcher<JobTask, AccountData[], AccountData>
         }
         break
       case TrieNodeFetcher:
+        // TODO update to check if heal phase completed successfully and then continue with next
```

Comment:
this function with this case would/should anyway be called when heal phase completed successfully